### PR TITLE
Fixes css of assignee suggestions input

### PR DESCRIPTION
**Pull request by Ajeyakrishna-k**
### Description:
- Updates the css assignee suggestions input in issues page and task card page
- Adds a default start date value in Task assignment form

### Dev Tested:
- [x] Yes

### Images/video of the change:
Dimensions: 

| Header 1 | Header 2 |
|----------|----------|
| Width | 330px | 
| Height | 390px |
| padding | 15px | 

Screenshot 2023-11-01 at 8 13 52 AM

Screenshot 2023-10-28

Screenshot 2023-10-28

Code coverage: 
Screenshot 2023-10-28 at 2 58 31 AM

### CODE REVIEW

File: src/components/issues/ActionForm.tsx [hunk 1]
```diff
@@ -135,10 +137,18 @@ const ActionForm: FC<ActionFormProps> = ({
                                 : styles.suggestions_container
                         }
                     >
+                        <label
+                            htmlFor="assignee"
+                            className={styles.assign_label}
```
Comment (ajoykumardas12) [inline src/components/issues/ActionForm.tsx:142]: Is className `assign_label` intended or a typo?

File: src/components/tasks/SuggestionBox/suggestion.module.scss [hunk 2]
```diff
@@ -43,14 +43,18 @@
...
 .suggestionsInput {
-    font-size: 1.3rem;
     font-weight: 400;
-    width: 7rem;
-    height: 2.5rem;
+    width: 100%;
     display: inline-block;
+    padding: 0 4px 0 4px;
+    height: 2rem;
+    border: 1px solid #1f1f1f;
```
Comment (ajoykumardas12) [inline src/components/tasks/SuggestionBox/suggestion.module.scss:55]: Should use variable for the color.

File: src/components/tasks/card/index.tsx [hunk 3]
```diff
@@ -603,7 +603,9 @@ const Card: FC<CardProps> = ({
                 </span>
                 {isEditable
                     ? isUserAuthorized && (
-                          <div className={classNames.assignedToSection}>
+                          <div
+                              className={`${classNames.assigneeSuggestionInput} ${classNames.assignedToSection}`}
```
Comment (ajoykumardas12) [inline src/components/tasks/card/index.tsx:607]: These class names should be using BEM convention used mostly in the app I think.

Noticed few components having different naming convention, which should be changed to BEM(out of scope of this PR).

File: src/components/issues/ActionForm.tsx [hunk 4]
```diff
@@ -135,10 +137,18 @@ const ActionForm: FC<ActionFormProps> = ({
                                 : styles.suggestions_container
                         }
                     >
+                        <label
```
Comment (ajoykumardas12) [inline src/components/issues/ActionForm.tsx:140]: Should there be little space between label and input? Currently they are touching each other.

File: src/components/tasks/SuggestionBox/suggestion.module.scss (hunk 2 above)
Comment (Ajeyakrishna-k) [reply src/components/tasks/SuggestionBox/suggestion.module.scss]: Done.

File: src/components/issues/ActionForm.tsx (hunk 1 above)
Comment (Ajeyakrishna-k) [reply src/components/issues/ActionForm.tsx:142]: Intended it has been used in several places in several different complements already.

File: src/components/issues/ActionForm.tsx (hunk 4 above)
Comment (Ajeyakrishna-k) [reply src/components/issues/ActionForm.tsx:140]: Are you referring to code or UI?

Comment (Ajeyakrishna-k) [reply src/components/issues/ActionForm.tsx:140]: Indentation is taken care by prettier.

Comment (ajoykumardas12) [reply src/components/issues/ActionForm.tsx:140]: I was referring to UI. The 'g' in 'Assignee' label seems to be touching the input in screenshot?

Comment (Ajeyakrishna-k) [reply src/components/issues/ActionForm.tsx:140]: Added slight gap.

**Review by ajoykumardas12 — APPROVED** [review]

**Review by ardourApeX — APPROVED** [review]

**Review by Achintya-Chatterjee — APPROVED** [review]
:rocket:

**Pull request merged**